src/components/__internal__/popup/popup.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
Let's move Popup our of internal. I think it's a component we should be encourage for re-use -- it can be reused in Tooltips, ContextMenus, etc.